tests/test_comparison_template_lib.py

```diff
@@ -162,3 +163,118 @@ def test_date_comparison_error_logger(ctl):
     # Metric len == 0
     with pytest.raises(ValueError):
         ctl.date_comparison("date", datediff_thresholds=[1], datediff_metrics=[])
+
+
+## name_comparison
+
+
+@pytest.mark.parametrize(
+    ("ctl"),
+    [
+        pytest.param(ctld, id="DuckDB Name Comparison Simple Run Test"),
+        pytest.param(ctls, id="Spark Name Comparison Simple Run Test"),
+    ],
+)
+def test_name_comparison_run(ctl):
+    ctl.name_comparison("date")
```

Comment:
possibly slightly confusing choice for the column name here

Reply:
Yep, stupid - will fix